src/site/src/lib/footer/Footer.svelte

```diff
@@ -32,7 +32,7 @@
 								<li><a href="https://opentelemetry.io">Official website</a></li>
 								<li>
 									<a
-										href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md"
+										href="https://opentelemetry.io/docs/specs/otel/"
```

Comment:
Or perhaps directly to https://opentelemetry.io/docs/specs/otel/overview instead of https://opentelemetry.io/docs/specs/otel?

```suggestion
										href="https://opentelemetry.io/docs/specs/otel/overview/"
```

Reply:
I think `/otel` is better, as we refer here to the spec.